app/helpers/application_helper.rb

```diff
@@ -23,7 +23,7 @@ def sortable(column)
   def whodunnit(object)
     if object.versions.last&.event == "update"
       "Last updated on #{l object.versions.last.created_at.localtime} by #{User.where('id = ?', object.versions.last.whodunnit).first.email}"
-    elsif object.versions.last&.event == "create"
+    elsif object.versions.last&.event == "create" && object.versions.last.whodunnit != nil
```

Comment:
There's never a need to do `== nil` or `!= nil` in Ruby. If falsiness is enough, then just:

```
!object.versions.last.whodunnit
```

If we need a "hard" check for `nil`:

```
!object.versions.last.whodunnit.nil?
```